src/core/test/system/lender/assessor/default_interest.sol

```diff
@@ -17,7 +17,7 @@ pragma solidity >=0.5.12;
 
```

Comment:
Dont look at this files. Nothing changed in interest tests.

I realized i switched the names in the test files.

`default_interest.sol` had the tests for Full invest interest
`full_invest_interest.sol` had the tests for default interest